### PR TITLE
Prize pool rebalance for arcade machines

### PR DESCRIPTION
**Pull request by deathrobotpunch**
# About the pull request
adds a fuck ton of cool and fun prizes to arcade machines that you can win!

# Explain why it's good for the game
IMG_2343

I updated the list massively so there is an actual reason to play arcade machines, with the old prizes there was no incentive as there was no fun things to win.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: more items have been added to the arcade machine prize pool.
/:cl:

### CODE REVIEW

**Comment by Blundir** [comment]
Remove the OWLF mask please, putting factional items in a jokingly isn't a good idea

**Comment by cuberound** [comment]
oh god, the bikehorn, I will play the hell out of arcades to get my hands on it as scout

**Comment by deathrobotpunch** [comment]
> Remove the OWLF mask please, putting factional items in a jokingly manner isn't a good idea, also you've structrued pr description that way, so before and after can be easily confused

fine I’ll replace it with headbands!

**Comment by deathrobotpunch** [comment]
OK so I found the gimmick clothes category and I want to ask, can I add the fucking rambo pants?

**Comment by Blundir** [comment]
> OK so I found the gimmick clothes category and I want to ask, can I add the fucking rambo pants? also I edited the PR so it’s more readable

Gimmick clothing is intended for use during halloween

**Comment by deathrobotpunch** [comment]
> Gimmick clothing is intended for use during halloween

got it, thanks!

**Review by kiVts — APPROVED** [review]

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -15,16 +15,39 @@
...
+							/obj/item/toy/plush/bee = 2,
+							/obj/item/toy/plush/moth = 2,
+							/obj/item/toy/plush/rock = 5,
+							/obj/item/storage/box/snappops = 5,
+							/obj/item/facepaint/lipstick = 1,
+							/obj/item/facepaint/lipstick/jade = 1,
+							/obj/item/clothing/head/headband/gray = 5,
+							/obj/item/clothing/head/headband/red = 5,
+							/obj/item/clothing/mask/tornscarf = 5,
+							/obj/item/clothing/shoes/stompers = 1
```
Comment (Blundir) [inline code/game/machinery/computer/arcade.dm:49]: I've made a special box for these shoes, so you might want to add that instead

**Comment by efzapa** [comment]
if you know the code, maybe add charge cards with like 10-20 bucks on them, good prisoner content since they otherwise cant buy anything without an ID, but can with a charge card

**Comment by deathrobotpunch** [comment]
sure

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -15,16 +15,39 @@
...
```
Comment (Blundir) [inline code/game/machinery/computer/arcade.dm:49]: I'll make this more clear, as a person who added them in the first place, i don't think they fit in this list logically, remove them

**Review by Blundir — DISMISSED** [review]

**Comment by efzapa** [comment]
you need an intendation after the comma on the wallet item

don't think it'll work if you just use one line or no proper indents

**Comment by Nanu308** [comment]
Current rewards on the PR list seems fine to me now with the changes made per request of other maintainers

**Pull request merged**